src/main/java/teammates/sqllogic/api/Logic.java

```diff
@@ -49,6 +55,17 @@ public Course createCourse(Course course) throws InvalidParametersException, Ent
         return coursesLogic.createCourse(course);
     }
 
+    /**
+     * Get usage statistics within a time range.
+     */
+    public List<UsageStatistics> getUsageStatisticsForTimeRange(Instant startTime, Instant endTime) {
+        assert startTime != null;
+        assert endTime != null;
+        assert startTime.isBefore(endTime);
```

Comment:
Move these assertions to the actual logic class. Trying to keep the maintenance cost of this facade class to a minimum.